salt/edx/run_ansible.sls

```diff
@@ -97,19 +97,24 @@ run_ansible:
     - env:
         HOME: /root
 
-update_max_upload_size_for_lms:
+update_max_upload_size_and_set_csrf_for_lms:
   file.replace:
     - name: /etc/nginx/sites-enabled/lms
     - pattern: 'client_max_body_size\s+\d+M;'
     - repl: 'client_max_body_size {{ salt.pillar.get("edx:edxapp:max_upload_size", "20") }}M;'
     - backup: False
     - require:
         - cmd: run_ansible
+  file.line:
```

Comment:
This won't work as it's 2 different `file` module statements with the same ID. It will need to be split into a separate call.